### ISSUE TITLE
req.param() audit

### ISSUE THREAD

**Issue by mmmaaatttttt** [rithmschool/numbers_api#20]
In express v2, you could use a function `req.param()` to access data coming from either the url or the query string. This was deprecated and now you just access `req.params` or `req.query` directly. however, in the express version bump i made for the project i just automatically changed `req.param()` to `req.params`, even if the data was coming from the query string. There may be bugs because of this, let's audit every place we reference `req.params` and be sure that we're trying to use a req parameter and not something else (e.g. a query string parameter)

**Comment by stephanedreyfus** [comment]
👍

**Comment by mmmaaatttttt** [comment]
@ngjamesng @richarddowdy do y'all want to take a look at this? it's possible the app is fine and there are no changes to be made.

**Comment by ngjamesng** [comment]
I took a quick look and tried out a couple of query strings as the instructions say on the README.md/web page. It appears the express v2 => express v4 update had some breaking changes. There are some lines that should be changed in the codebase to use a `req.query.XXX` if we were to preserve the behavior of how the README.md currently instructs. 

I'll write down some of the query options we should pay attention to. During my initial pass, we need to find and update the following:
• `req.params.write` => `req.query.write`
• `req.params.callback` => `req.query.callback` 

I've tried the query strings and they seem to work, however it was just a quick passthrough. @richarddowdy did you find any other breaking req.params?
